package.json

```diff
@@ -1,55 +1,65 @@
 {
-  "name": "dockercomposator-plugin",
+  "name": "docker-composator-plugin",
   "version": "0.1.0",
-  "description": "DockerCompose plugin for Leto-Modelizer.",
-  "main": "dist/dockercomposator-plugin.js",
-  "exports": "./dist/dockercomposator-plugin.js",
+  "description": "Plugin for creating docker compose files",
+  "main": "dist/docker-composator-plugin.js",
+  "exports": "./dist/docker-composator-plugin.js",
   "scripts": {
     "build": "webpack",
     "build:docs": "jsdoc --readme README.md --package package.json -r ./src/ -c jsdoc.config.json -d docs",
-    "lint": "eslint --max-warnings=0 --ext .js src",
-    "lint:fix": "eslint --max-warnings=0 --fix --ext .js src",
-    "lint:report": "eslint --max-warnings=0 --ext .js src -f json-relative > eslint.json",
+    "lint": "eslint --ext .js src",
+    "lint:fix": "eslint --fix --ext .js src",
+    "lint:report": "eslint --ext .js src -f json-relative > eslint.json",
     "prepare:docs": "sed -i 's#taffydb#@jsdoc/salty#g' node_modules/better-docs/publish.js",
     "test": "jest",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "generate:parser": "node scripts/generate_parser.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ditrit/dockercomposator-plugin.git"
   },
   "keywords": [
     "Leto-modelizer",
-    "dockercomposator-plugin",
-    "docker-compose"
+    "docker-compose",
+    "docker-composator-plugin"
+  ],
+  "authors": [
+    "Yassine Chamkhi <yessinechamkhi@gmail.com>",
+    "Mohamed Rayen Romdhane <mohamedrayenromdhane21@gmail.com>",
+    "Emna Belhaj <emnabelhaj98@gmail.com>"
   ],
-  "author": "Vincent Moittie <moittie.vincent@gmail.com>",
   "license": "Mozilla Public License 2.0",
   "bugs": {
     "url": "https://github.com/ditrit/dockercomposator-plugin/issues"
   },
   "homepage": "https://github.com/ditrit/dockercomposator-plugin#readme",
   "dependencies": {
-    "leto-modelizer-plugin-core": "github:ditrit/leto-modelizer-plugin-core#0.17.0"
+    "js-yaml": "^4.1.0",
```

Comment:
All dependency modification have to be in another commit:
- install library X, Y, Z
- update dependencies -> this commit is at the end of history